apps/apps.humanatlas.io/src/app/pages/us6/embed-sidenav-content/embed-sidenav-content.component.ts

```diff
@@ -41,4 +45,9 @@ export class EmbedSidenavContentComponent {
   openDocumentLink() {
     window.open(this.documentLink(), '_blank');
   }
+
+  /** Trigerred when user clicks on the Copy button */
+  openSnackbar() {
```

Comment:
Rename to `openCopiedSnackbar` to indicate what the snackbar is actually for (`openSnackbar` is simply to generic)